src/DynamoCore/Graph/Workspaces/PackageInfo.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Graph.Workspaces
+{
+    /// <summary>
+    /// Class containing info about a Dynamo package. 
+    /// Used for serialization.
+    /// </summary>
+    internal class PackageInfo
```

Comment:
does this same class already exist in another namespace? Just want to make sure we don't have two types with the same name.

Reply:
We have a `PackageFileInfo` class. https://github.com/DynamoDS/Dynamo/blob/ec10f936824152e7dd7d6d019efdcda0d78a5264/src/DynamoPackages/PackageFileInfo.cs

I'm going to rename this new class `PackageDependencyInfo` for more clarity.